Header header
BoundingBox[] faces
Shirt[] shirts